cloud-service-providers/google-cloud/vertexai/python/airgap/docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+services:
+  nim-llama-8b:
+    image: nvcr.io/nim/meta/llama3-8b-instruct:latest
```

Comment:
Can we pin this to a version rather than latest?

Reply:
Will change to 1.0.0

Reply:
For my learning could I know the reason?

Reply:
In general we want to pin the versions to something that has been tested and is known to work.

If we put the tag here as :latest, a year from now there may be breaking API changes and this example may no longer work if it was relying on something in the 1.0.0 container version.